### DISCUSSION TITLE
Sorry I am clueless, where to go now to read the Unturned Docs information?

### DISCUSSION THREAD

**Discussion by Ongjming** [SmartlyDressedGames/U3-Docs#240] [Q&A] [answered]
Everything was still here the day before, was surprised to find that everything is in codes now and the previous information is nowhere to be found.
Is everything chucked to the wiki or something?

Sorry, I am completely clueless so pardon my question even if it could seem obvious on where to find the information now.

**Comment by Ongjming** [comment, marked as answer]
I found this link: https://docs.smartlydressedgames.com/en/stable/
The UI is a bit confusing, but I guess I will roam around within that to try and find what I need. Thanks.

**Reply by MoltonMontro** [reply]
Don't be intimidated -- if you're having trouble navigating the table of contents on the left side (most stuff is alphabetical, similar to before), there's a search bar at the top-left.

**Reply by Ongjming** [reply]
Thanks for the encouragement and advice! :)
